### PR TITLE
 (README.MD) Update Invalid Links. Proper crediting.

### PR DESCRIPTION
**Pull request by StarryRin**
title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,14 +70,14 @@
 > 
 > - Reference: Modify settings menu
 > 
-### :star: [TOH+](https://github.com/SkullCreeper/TownOfHostPlus)
+### :star: [TOH+](https://github.com/ItzLoonie/TownOfHostPlus)
```
Comment (Ultradragon005) [inline README.md:73]: WHAT 
image

Comment (Ultradragon005) [reply README.md:73]: wait, these are probably not even used now due to the new update re-hauling the entire thing

**Review by HyperAtill — CHANGES_REQUESTED** [review]
TOHE Alchemist is not from Lotus

**Comment by KingPanda360** [comment]
> TOHE Alchemist is not from Lotus

Tealeaf was the one who gave the idea of alchemist a year ago. Plus marg ported it from project lotus.

**Comment by KingPanda360** [comment]
I messed up, it was SolarFlare who ported it. Not marg.

**Comment by HyperAtill** [comment]
> > TOHE Alchemist is not from Lotus
> 
> Tealeaf was the one who gave the idea of alchemist a year ago. Plus marg ported it from project lotus.

Oh.... okay I didn't know that
Sorry
Thanks

**Comment by KingPanda360** [comment]
> > > TOHE Alchemist is not from Lotus
> > 
> > Tealeaf was the one who gave the idea of alchemist a year ago. Plus marg ported it from project lotus.
> 
> Oh.... okay I didn't know that
> Sorry
> Thanks

You're Welcome

**Comment by Dxgamer7405** [comment]
https://github.com/user-attachments/assets/eb647a23-e167-4e5f-9857-e0ec474e2f9a

**Comment by Gurge44** [comment]
What are you all talking about?
Tealeaf gave the idea
**I** coded it into EHR (TOHE+ at the time)
Loonie ported it from EHR (TOHE+)

I literally remember making a pr for him because he forgot to call 3 methods and just complained about the role not working

**Comment by Ultradragon005** [comment]
💀

**Comment by Gurge44** [comment]
SolarFlare did port Alchemist from PL, but that never made it into .... TOHER? TOHRE? TOHEN? TOHE? whatever, it was only in their version (SLE (SolarLoonieEdit))

**Comment by MargaretTheFool** [comment]
> I messed up, it was SolarFlare who ported it. Not marg.

sylveon pfp = me, trust (don't trust I'm lying)

**Pull request merged**